contracts/Platform.sol

```diff
@@ -109,6 +115,11 @@ contract Platform is Ownable, VRFConsumerBaseV2 {
     vrfCoordinator = VRFCoordinatorV2Interface(_vrfCoordinator);
     subscriptionId = _subscriptionId;
     keyHash = _keyHash;
+
+    // Default value: 0.1 eth / 30 days / 24 hours / 60 minutes
+    rewardForPlayedMinute = 2314814814814;
```

Comment:
Note that this is only a default value, so the reward is not empty upon deploying the smart contract and before running a task to set the intended reward amount.